Detectors/TRD/base/src/TrackletTransformer.cxx

```diff
@@ -96,22 +112,23 @@ float TrackletTransformer::calculateDy(int slope, double lorentzAngle, double dr
   // NOTE: check what drift height is used in calibration code to ensure consistency
   // NOTE: check sign convention of Lorentz angle
   // NOTE: confirm the direction in which vDrift is measured/determined. Is it in x or in direction of drift?
-  double lorentzCorrection = TMath::Tan(lorentzAngle) * mXAnode;
+  double lorentzCorrection = TMath::Tan(exb) * mXAnode;
 
   // assuming angle in Bailhache, fig. 4.17 would be positive in our calibration code
   double calibratedDy = rawDy - lorentzCorrection;
 
   return rawDy;
```

Comment:
You could even return `calibratedDy` here. In the default calib object I put ExB to zero, so the correction does not have an an effect anyways.